install.sh

```diff
@@ -3495,6 +34,14 @@ execute_installation() {
 
     load_install_dir
 
+    # Clone the repository to the install folder
+    # and run the install.sh from there.
+    if ! is_running_internal_install && ! is_update_only; then
```

Comment:
What's this block for, when would this `install.sh` not run from the clone repo?
On the first install from the old/current way?

Reply:
This if is only entered when:
- we are not an update and
- we run the script from lets say `curl/wget` (externally not inside the release clone which is the ground truth)

Reply:
Please close this, if this answers your question.